cmd/create/oidcconfig/cmd.go

```diff
@@ -112,21 +113,58 @@ func run(cmd *cobra.Command, argv []string) {
 		}
 	}
 
+	oidcConfigInput := buildOidcConfigInput(r)
 	oidcConfigStrategy, err := getOidcConfigStrategy(mode)
 	if err != nil {
 		r.Reporter.Errorf("%s", err)
 		os.Exit(1)
 	}
-	bucketUrl, privateKeyFilename := oidcConfigStrategy.execute(r)
+	oidcConfigStrategy.execute(r, oidcConfigInput)
 	if r.Reporter.IsTerminal() {
 		r.Reporter.Infof("To create a cluster with this oidc config, please run:\n"+
 			"rosa create cluster --sts --oidc-endpoint-url %s "+
-			"--bound-service-account-signing-key-path ./%s", bucketUrl, privateKeyFilename)
+			"--bound-service-account-signing-key-secret-name %s",
```

Comment:
This should be `--bound-service-account-signing-key-secret-arn`, also why is it templated to `oidcConfigInput.BucketName`?

Also, do you think there's a better name for this flag? Like `--oidc-private-key`, for example?
cc @jharrington22

Reply:
Sure, will do!

Reply:
Oh this is leftover, this time I can't have this output actually. I can do it for auto mode, however in manual the user will have to run the commands which will then output the correct ARN. And to keep things consistent I'd rather not output this